lib/sensors/CMakeLists.txt

```diff
@@ -27,3 +27,8 @@ add_library(QEI STATIC)
 target_sources(QEI PRIVATE src/QEI.cpp)
 target_include_directories(QEI PUBLIC include)
 target_set_mbed_dependency(QEI)
+
+add_library(UltraSonicSensor STATIC)
```

Comment:
Ultrasonic is a single word so it should be `UltrasonicSensor`. pls fix in all files/folders/classes etc.